CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ read the notes of the `Enso 2.0.0-alpha.1` release.
 - [Assigning intermediate expressions to values][1067]. Nodes added with searcher will have their 
   values automatically assigned to a newly generated variables. This allows Enso Engine to cache
   intermediate values, improving visualization performance.
+- [Not adding spurious imports][1209]. Fixed case where IDE would generate spurious import 
+  declarations when inserting method calls.
```

Comment:
Could you describe the issue a little bit more using simpler words and maybe bigger description? The changelog is for users snd they should understand what it is about. Even "inserting method calls" is not really understandable fully, as in the GUI its not obvious to every user what that's mean.

I would go this way (written fast, contains errors, just a scratch description):
```
[Not adding unnecessary library imports][1209]. Fixed situation when IDE was adding unnecessary libraries imports when selecting hints from node searcher. This makes the generated textual code easier to read.
```